src/app/projects/services/projects.service.ts

```diff
@@ -15,6 +15,15 @@ interface Filters {
 
 @Injectable()
 export class ProjectsService implements OnDestroy {
+  static allowedLocations = [
```

Comment:
just curious -- why did you move these into the service from the component?

Reply:
Because the service does the searching too and knows what things are searchable so it acts as a source of truth for projects